quickwit/quickwit-query/src/elastic_query_dsl/range_query.rs

```diff
@@ -76,6 +75,15 @@ impl ConvertableToQueryAst for RangeQuery {
                 (None, Some(lte)) => Bound::Included(lte),
                 (None, None) => Bound::Unbounded,
             },
+            format: match format {
+                None => None,
+                Some(v) => {
+                    match v {
+                        JsonLiteral::String(s) => Some(s),
+                        _ => None
+                    }
+                }
+            },
```

Comment:
at the moment I came up with the most straightforward solution, I wonder if that's something that you more or less expected to see @fulmicoton?